Documentation/CRDs/Block-Storage/ceph-block-pool-rados-namespace-crd.md

```diff
@@ -49,3 +49,40 @@ If any setting is unspecified, a suitable default will be used automatically.
 ### Spec
 
 - `blockPoolName`: The metadata name of the CephBlockPool CR where the rados namespace will be created.
+
+
+### Consuming StorageClass
+
+Once the RadosNamespace is created, a rbd-based StorageClass can be created to
+consume exactly this Namespace. For this purpose, the  `ClusterID value of the
+Radosnamespace needs to be put into the `ClusterID` field of the StorageClass
+spec  which usually holds the kubernetes namespace  hosting the rook-ceph
+cluster.
+
+## Extracting the ClusterID
+
+ ```shell
+ $ kubectl -n rook-ceph get cephblockpoolradosnamespace -o yaml
+
+  status:
+    info:
+      clusterID: 80fc4f4bacc064be641633e6ed25ba7e
+```
+
+## Defining the StorageClass
```

Comment:
I'd suggest no subheaders. The one heading for "Creating a Storage Class" seems sufficient for the full section.
```suggestion
```